aws/createResources.sh

```diff
@@ -1,7 +1,8 @@
 #!/usr/bin/env bash
 
 ROOT_NAME=DevDay2
-BUCKET_NAME=budilov-$ROOT_NAME
+# Bucket name must be all lowercase, and start/end with lowecase letter or number
+BUCKET_NAME=budilov-${ROOT_NAME,,}
```

Comment:
$ROOT_NAME  is a variable and it would resolve to DevDay2 -- which shell are you running this script in? This same command works for me.